test/pushito_test.exs

```diff
@@ -2,7 +2,30 @@ defmodule PushitoTest do
   use ExUnit.Case
   doctest Pushito
 
-  test "our unique test for now" do
-    assert 1 + 1 == 2
+  setup_all do
+    {:ok, _} = Application.ensure_all_started(:pushito)
+
+    on_exit fn ->
+      Application.stop(:pushito)
+    end
+
+    :ok
+  end
+
+  test "make a connection to APNs with Provider Certificates" do
+    cert_file = Application.fetch_env!(:pushito, :cert_file)
+    key_file = Application.fetch_env!(:pushito, :key_file)
+    apple_host = Application.fetch_env!(:pushito, :apple_host)
+    connection_name = :connection1
+    config = %Pushito.Config{:name => connection_name,
+                             :type => :cert,
+                             :apple_host => apple_host,
+                             :cert_file => cert_file,
+                             :key_file => key_file}
+
+    {:ok, pid} = Pushito.connect(config)
+    ^pid = Process.whereis connection_name
```

Comment:
Why not to use asserts  macros?? For example: `assert Process.whereis connection_name == pid`